cdisc_rules_engine/operations/variable_exists.py

```diff
@@ -4,4 +4,5 @@
 class VariableExists(BaseOperation):
     def _execute_operation(self):
         # get metadata
-        return self.params.target in self.params.dataframe
+        dataframe = self.data_service.get_dataset(self.params.domain)
```

Comment:
Negative case seems to work correctly, but I can't get the positive case to work by renaming VSNOTEST to VSTESTCD in dataset.json. It still says $testcd_exists=False

The other parts of the rule appear to work.

Reply:
Fixed with variable_exists operator update